therapy/therapy-blog.html

```diff
@@ -0,0 +1,300 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1" crossorigin="anonymous">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital@0;1&family=Quicksand:wght@300&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="../css/therapy.css">
+    <link rel="icon" href="../img/head-work.png">
+    <title>Maria Hope Ph.D.</title>
+</head>
+<body>
+    <header class="container" id="top">
+        <div class="row navigation">
+            <button type="button" class="btn btn-primary col-7 offset-1 cta">
+                <a href="therapy-book.html">Book an Appointment</a>
+            </button>
+            <div class="hamburger col-3 offset-1 text-right open-close" onclick="openNav()">
+                <a href="#">&#9776;</a>
+            </div>
+            <nav class="navbar navbar-expand-sm bg-custom open-sidenav" id="open">
+                <ul class="navbar-nav">
+                    <li class="text-right close-nav">
+                        <a href="#" class="close-div nav-link text-right open-close" onclick="closeNav()">&times;</a>
+                    </li>
+                    <li class="nav-item active overlay">
+                        <a href="therapy.html" class="nav-link">Home</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-about.html" class="nav-link">About</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-services.html" class="nav-link">Services</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-blog.html" class="nav-link">Blog</a>
+                    </li>
+                    <div class="divider"></div>
+                    <li class="nav-item">
+                        <a href="therapy-contact.html" class="nav-link">Contact</a>
+                    </li>
+                </ul>
+            </nav>
+            <div>
+                <h3 class="headings headings-therapy"><a href="therapy.html">Maria Hope, Ph.D. - Psychologist</a></h3>
+            </div>
+        </div>
+    </header>
+    <section class="container">
+        <div class="row">
+            <div class="section-intro">
```

Comment:
As per comments above, this element as well as 'blog-articles' below should have a column class applied to them as they are direct children of 'row'